src/System.ServiceModel.Http/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,223 @@
+    <Left>lib/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>lib/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.HttpBindingBase.get_HostNameComparisonMode</Target>
+    <Left>lib/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>lib/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.HttpBindingBase.set_HostNameComparisonMode(System.ServiceModel.HostNameComparisonMode)</Target>
+    <Left>lib/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>lib/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.HttpTransportSecurity.ConfigureTransportProtectionAndAuthentication(System.ServiceModel.Channels.HttpsTransportBindingElement,System.ServiceModel.HttpTransportSecurity)</Target>
+    <Left>lib/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>lib/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.HttpTransportSecurity.get_Realm</Target>
+    <Left>lib/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>lib/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.HttpTransportSecurity.set_Realm(System.String)</Target>
+    <Left>lib/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>lib/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.NetHttpBinding.#ctor(System.String)</Target>
+    <Left>ref/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>ref/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.WSHttpBindingBase.set_EnvelopeVersion(System.ServiceModel.EnvelopeVersion)</Target>
+    <Left>ref/net8.0/System.ServiceModel.Http.dll</Left>
+    <Right>ref/net8.0/System.ServiceModel.Http.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.Channels.HttpTransportBindingElement.get_Proxy</Target>
+    <Left>ref/netstandard2.0/System.ServiceModel.Http.dll</Left>
```

Comment:
This looks like a genuine problem, this api is listed in the netstandard reference assembly but isn't public on .NET Framework, so should only be in our net8 reference assembly.